### PR TITLE
Implement Layout Component

### LINKED ISSUE #2: Create Layout component
## Description

Create a simple layout component that will wrap all of our application. We are using tailwindcss for styling, so make sure to familiarise yourself with [utility first approach](https://tailwindcss.com/docs/utility-first) to styling. Use a simple [tailwindcss container](https://tailwindcss.com/docs/container) for now. It's valuable to take a look at the tailwindcss docs deeper.

## Steps to take

1. Create a folder `/src/components/layout`
2. Create a file for the component `/src/components/layout/layout.tsx`
3. Implement the layout component:
    1. The component should be a functional react component of type `React.FC`, take a look at [`Example` component](https://github.com/vu-vibedosa/apsitvarkom-fe/blob/dev/src/components/Example.tsx#L8). Check differences between class and functional react components [here](https://www.twilio.com/blog/react-choose-functional-components)
    2. It should take in `children` through props. See [this blog post](https://blog.logrocket.com/using-react-children-prop-typescript/)
    3. Wrap the children prop in a [container](https://tailwindcss.com/docs/container) and center it when rendering
    4. Don't forget to export the functional component
4. In `App.tsx`, instead of a [simple div](https://github.com/vu-vibedosa/apsitvarkom-fe/blob/dev/src/App.tsx#L6), use your newly created `Layout` component

## Acceptance criteria

`Layout` component created that can be used something like this:
```jsx
<Layout>
    <div>Hello world</div>
    <MyOtherComponent/>
</Layout>
```

### PR DESCRIPTION
**Pull request by BaVilius**
Created a simple layout component which can take in children through Props.

### CODE REVIEW

File: src/components/layout/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const Layout: React.FC<Props> = ({ children }: Props) => {
+  return (
+    <>
+      <div className="container mx-auto">{children}</div>;
+    </>
```
Comment (BenasB) [inline src/components/layout/layout.tsx]: No need for a [`React.Fragment`](https://reactjs.org/docs/fragments.html#short-syntax), just return the `div` directly

File: src/components/layout/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const Layout: React.FC<Props> = ({ children }: Props) => {
```
Comment (BenasB) [inline src/components/layout/layout.tsx]: `({ children }: Props)` can be just `({ children })` since the type is inferred from `React.FC<Props>`

**Review by BenasB — CHANGES_REQUESTED** [review]
Sorry, I forgot to mention in the task that usually (what I've seen) component **files** are named in PascalCase, so let's stick to that convention (especially when I already created an example component with that naming). `src/components/layout/layout.tsx` -> `src/components/layout/Layout.tsx`

Small changes required, but overall pretty good, thanks

File: src/components/layout/layout.tsx (hunk 1 above)
Comment (BaVilius) [reply src/components/layout/layout.tsx]: You are right. Was playing around with React, forgot to remove.

File: src/components/layout/layout.tsx (hunk 2 above)
Comment (BaVilius) [reply src/components/layout/layout.tsx]: Alright

**Review by BenasB — APPROVED** [review]
LGTM, feel free to merge

**Pull request merged**